sierra_adapter/sierra_items_to_dynamo/src/main/scala/uk/ac/wellcome/platform/sierra_items_to_dynamo/dynamo/package.scala

```diff
@@ -0,0 +1,34 @@
+package uk.ac.wellcome.platform.sierra_items_to_dynamo.dynamo
+
+import com.gu.scanamo.DynamoFormat
+import com.gu.scanamo.syntax._
+import uk.ac.wellcome.models.transformable.sierra.{
+  SierraItemRecord,
+  SierraRecordNumber
+}
+import uk.ac.wellcome.storage.dynamo._
+import uk.ac.wellcome.storage.type_classes.IdGetter
+
+object dynamo {
+  implicit val recordNumberFormat =
+    DynamoFormat
+      .coercedXmap[SierraRecordNumber, String, IllegalArgumentException](
+        SierraRecordNumber
+      )(
+        _.withoutCheckDigit
+      )
+
+  implicit val idGetter: IdGetter[SierraItemRecord] =
```

Comment:
I know this is used by the VersionedDao, but it doesn't feel like this pckage is the right place for this. My personal preference would be to put it in the SierraItemRecord companion object

Reply:
mmh, but you would need the dependecy to scanamo wherever SierreItemRecord and SierraRecordNumber are defined...
Not too fussed about it but I'm not a big fan of these package objects. I think we should keep them to a minimum. They make things hard to find unless you know already where to look for them